src/main/java/com/google/opensesame/github/GitHubGetter.java

```diff
@@ -1,12 +1,18 @@
 package com.google.opensesame.github;
 
+import com.google.opensesame.util.RateLimitExceededException;
 import java.io.IOException;
+import java.net.HttpURLConnection;
 import org.kohsuke.github.GitHub;
 import org.kohsuke.github.GitHubBuilder;
+import org.kohsuke.github.RateLimitHandler;
 
 public final class GitHubGetter {
   private static GitHub gitHub = null;
 
+  /** See {@link GitHubRateLimitHandler} */
+  private static GitHubRateLimitHandler rateLimitHandler = new GitHubRateLimitHandler();
```

Comment:
nit: Knowing that `buildGitHubInterface` is only run once and the `rateLimitHandler` is only used in that function, it might be best to move this to a local variable in `buildGitHubInterface`.

Reply:
+1

Reply:
It's usually best to keep the scope of variables as limited as possible. This helps prevent them from being unintentionally accessed and modified.